src/briefcase/commands/convert.py

```diff
@@ -603,6 +617,13 @@ def build_gui_context(
         # already is set up for a GUI-framework, then those dependencies should already be listed.
         # To prevent the same dependency being listed twice (once in the PEP621-section and once in the
         # briefcase-section), possibly with different versions, we set the GUI-framework to None here.
+        if self.input_console_app(
+            override_value=project_overrides.pop("console_app", None)
+        ):
+            return {
+                "gui_framework": "None",
+                "pyproject_table_briefcase_app_extra_content": "\nconsole_app = true\n",
```

Comment:
If this is a feature we're going to rely on, we should probably add `console_app` as a feature in the [Briefcase template](https://github.com/beeware/briefcase-template) itself, rather than slipping it into "extra" content.

Reply:
alright, I will make a commit there first. should I open an issue first?
Thanks for the guidance.

Reply:
There's no need for a separate issue; you can use `Refs beeware/briefcase#1234` syntax to reference an issue or PR in a separate project.